internal/evaluator/conftest_evaluator.go

```diff
@@ -279,6 +279,18 @@ func (c conftestEvaluator) Evaluate(ctx context.Context, inputs []string) (Check
 	return results, nil
 }
 
+// Helper function to sort Result lists.
```

Comment:
Nice! I like it. Could this function be moved over to `internal/output/output.go`? It looks like it's only used there, so maybe it can be just a private function in that file now.

Reply:
@simonbaird That's actually how I originally wrote it, but putting it there means we'd need to export `extractStringFromMetadata()` from `conftest_evaluator.go` so it can be accessed from `internal/output/output.go`.

I could probably just replicate `extractStringFromMetadata()` in the function if we really want to minimize exports. But that (and tests) are a job for tomorrow :wink:

Reply:
I think sorting in the `internal/output` is the way to go, never mind the duplication of `result.Metadata["code"]`

Reply:
Personally I don't mind, whichever feels right.

Reply:
When I add in sorting via `term` as well (in the revision I just pushed), I end up using the `extractStringFromMetadata()` function enough that I think it's worth exporting, IMO.